### ISSUE TITLE
Weapon Glitch

### ISSUE THREAD

**Issue by HytroN** [XxFri3ndlyxX/esx_ati#1]

**Comment by XxFri3ndlyxX** [comment]
Yes if you put 200 and you type 1 to remove, 199 will be in the trunk.

**Comment by HytroN** [comment]
Yea it will mean there will be 200 pistols in the trunk.

That's my point

**Comment by XxFri3ndlyxX** [comment]
Well unless you know a way, i do not believe it is possible to do what you are asking. Because munition are tied to a category of weapon.

**Comment by HytroN** [comment]
Can you provide your discord, so it makes it easier to communicate?

**Comment by XxFri3ndlyxX** [comment]
I am sorry but i cannot give you my discord as i do not add anyone unless i know them from a long time or they are real life friends. Altho i could make a discord server dedicated for my github. But not right now as i just woke up and gonna chill drink a coffee and eat.

**Comment by HytroN** [comment]
Alright, no problem. But to solve this I think, the weapon weight should be equal as the car trunk weight, that means you only can put 1 in and take 1 out. This will fix it because the ammo should stay saved on you when you pick it up again.

**Comment by XxFri3ndlyxX** [comment]
You can try this and let me know how it goes. But i believe once you put 1 "weapon" it will put one weapon with one munition while you'll still have one weapon on you.  I will have a look on native later one and see if anything can be done.

**Comment by HytroN** [comment]
Sounds good! Keep the good work up!
